codegen/src/main/java/software/amazon/awssdk/codegen/naming/DefaultNamingStrategy.java

```diff
@@ -272,7 +272,7 @@ public String getEnumValueName(String enumValue) {
         String result = enumValue;
 
         // Special cases
-        result = result.replaceAll("textORcsv", "TEXT_OR_CSV");
+        result = result.replace("textORcsv", "TEXT_OR_CSV");
```

Comment:
Is this safe?

Reply:
yup, `replaceAll` is for regex, if chars then `replace` is less expensive